orcid-web/src/main/webapp/static/javascript/script.js

```diff
@@ -129,24 +129,31 @@ var OrcidCookie = new function() {
     };
 };
 
-var _gaq = _gaq || [];
-
 var OrcidGA = function() {
     // test and make sure _gaq is working. disconnect.me chrome plugin has
     // caused silent _gaq failures. This check allows us to detect that
     // situation
     var gaEnabled = false;
```

Comment:
Hey @rcpeters,

But, according to the link we saw yesterday: 

http://davidwalsh.name/detecting-google-analytics

The way to test that ga was loaded is like this: 

https://github.com/ORCID/ORCID-Source/pull/2353/files#diff-97eba34be435b164fc1d5c488a1edd13R137

So, you want me to make that a function (like onLoad) and call it when the OrcidGA is loaded?

Thanks